spec/ApplicationPattern.yaml

```diff
@@ -2014,6 +2014,7 @@ paths:
                     find [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-c-*/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/release-number]'
                 future-application-name:
                   type: string
```

Comment:
Similar to /v1/update-client, /v1/bequeath-your-data-and-die :: new-application-name also updates the application-name in load file. Shall we also add the pattern to the attribute to ensure the new-application-name is a valid value. 
Since, this value shall be used in multiple callbacks, this constraint would be helpful. 
Kindly let me know your views.

Reply:
The problem should already be solved by adding the pattern to the RO://register-application.
I would like to keep the number of occasions of the pattern definition low.

Reply:
I too have the same understanding regarding this decision. 
Since , /v1/update-client has been included with the pattern, I got a second thought on that decision like we included pattern while updating the application-name value. 

Query : 
Please let me know if we can also remove the pattern from /v1/update-client, since it is a callback of /v1/relay-server-replacement (which is triggered after /v1/bequeath-your-data-and-die). 
Or please let me know if I am missing any point here. Sorry for the trouble.

Reply:
Removed pattern from /v1/update-client :: future-application-name